app/models/maths_and_physics/admin_checks_presenter.rb

```diff
@@ -1,4 +1,11 @@
 module MathsAndPhysics
+  # Used to display the information a claim checker needs to check to either
+  # approve or reject a claim
+  #
+  # Note this presenter is only intented for use with eligible claims and
+  # therefor makes certain assumptions about the claim and eligibility.
```

Comment:
"therefore"

Reply:
This documentation gets removed in a later refactor so maybe we can let it slide 😬

Reply:
:+1: